src/components/Button/PButton.vue

```diff
@@ -24,32 +27,31 @@
   import { RouteLocationRaw } from 'vue-router'
   import PIcon from '@/components/Icon/PIcon.vue'
   import PLoadingIcon from '@/components/LoadingIcon/PLoadingIcon.vue'
-  import { Icon } from '@/types/icon'
-  import { Size } from '@/types/size'
+  import { Icon, ButtonKind } from '@/types'
   import { isRouteExternal } from '@/utilities/router'
 
-  type ButtonClass = 'primary' | 'secondary' | 'inset' | 'flat' | 'danger' | 'danger-secondary'
 
   const props = defineProps({
-    secondary: Boolean,
-    inset: Boolean,
-    flat: Boolean,
-    danger: Boolean,
-    rounded: Boolean,
-    disabled: Boolean,
-    loading: Boolean,
-    size: {
-      type: String as PropType<Size>,
-      default: 'md',
+    kind: {
+      type: String as PropType<ButtonKind>,
+      default: 'default',
     },
     icon: {
       type: String as PropType<Icon>,
       default: undefined,
     },
+    iconAppend: {
+      type: String as PropType<Icon>,
+      default: undefined,
+    },
     to: {
       type: [String, Object] as PropType<RouteLocationRaw>,
       default: undefined,
     },
+    small: Boolean,
+    activated: Boolean,
```

Comment:
What is `activated`?

Reply:
So the `activated` variant is really only for default buttons when they're part of a button group.
<img width="360" alt="image" src="https://github.com/PrefectHQ/prefect-design/assets/6776415/0d0b425b-406e-414e-812f-693df2717464">

I could probably just apply this as overrides from the button group though, huh..

Reply:
hmm, can this be a separate `kind` rather than its own prop? Then I think this make sense. I'd just make it `active` as the kind.

Reply:
I removed it for now, will reintroduce it when I hit up button groups.. probably in this same PR but after. Might try and do it from that component anyway.